roles/splunk_spark_master/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Set privilege escalation user"
+  set_fact:
+    privileged_user: "{% if ansible_system is match('CYGWIN*|Win32NT') %}Administrator{% else %}root{% endif %}"
```

Comment:
I don't think you need to define `privileged_user` here - it should already be available because you have `roles/splunk_spark_master/meta/main.yml` with a dependency on `splunk_common`.

Reply:
You are correct, I added this in before I included the splunk_common dependency and forgot to remove it.